pkg/controller/pravegacluster/pravegacluster_controller_test.go

```diff
@@ -714,5 +714,186 @@ var _ = Describe("PravegaCluster Controller", func() {
 				})
 			})
 		})
+
+		Context("Custom spec with ExternalAccess with annotations and empty domain name", func() {
+			var (
+				client client.Client
+				err    error
+			)
+
+			BeforeEach(func() {
+				annotationsMap := map[string]string{
+					"service.beta.kubernetes.io/aws-load-balancer-access-log-s3-bucket-prefix": "abc",
+				}
+				p.Spec = v1alpha1.ClusterSpec{
+					Version: "0.3.2-rc2",
+					ExternalAccess: &v1alpha1.ExternalAccess{
+						Enabled:    true,
+						Type:       corev1.ServiceTypeClusterIP,
+						DomainName: "",
+					},
+					Pravega: &v1alpha1.PravegaSpec{
+						SegmentStoreReplicas:            3,
+						SegmentStoreExternalServiceType: corev1.ServiceTypeLoadBalancer,
```

Comment:
There is no need to have `SegmentStoreExternalServiceType` feild here. This has already been tested in an earlier test. Please remove this field and related assertions.

Reply:
done